bomber.py

```diff
@@ -278,7 +278,7 @@ def selectnode(mode="sms"):
         check_for_updates()
         notifyen()
 
-        max_limit = {"sms": 500, "call": 15, "mail": 200}
+        max_limit = {"sms": 500000, "call": 150000, "mail": 200000}
```

Comment:
The values are set to the ones provided for a valid reason. You cannot amp them up randomly like this.

:-1: to this.